scipy/sparse/linalg/isolve/tfqmr.py

```diff
@@ -0,0 +1,178 @@
+import warnings
+import numpy as np
+from .utils import make_system
+
+
+__all__ = ['tfqmr']
+
+
+def tfqmr(A, b, x0=None, tol=1e-5, maxiter=None, M=None,
+          callback=None, atol=None, show=False):
+    """
+    Use Transpose-Free Quasi-Minimal Residual iteration to solve ``Ax = b``.
+
+    Parameters
+    ----------
+    A : {sparse matrix, ndarray, LinearOperator}
+        The real or complex N-by-N matrix of the linear system.
+        Alternatively, `A` can be a linear operator which can
+        produce ``Ax`` using, e.g.,
+        `scipy.sparse.linalg.LinearOperator`.
+    b : {ndarray}
+        Right hand side of the linear system. Has shape (N,) or (N,1).
+    x0 : {ndarray}
+        Starting guess for the solution.
+    tol, atol : float, optional
+        Tolerances for convergence, ``norm(residual) <= max(tol*norm(b-Ax0), atol)``.
+        The default for `tol` is 1.0e-5
+        The default for `atol` is ``tol * norm(b-Ax0)``.
+
+        .. warning::
+
+           The default value for `atol` will be changed in a future release.
+           For future compatibility, specify `atol` explicitly.
+    maxiter : int, optional
```

Comment:
Maxiter is calculated below as `min(10000, 2*ndofs)`.  Perhaps worth mentioning the default here.

Reply:
Thank you, I will add the the default description here.